server/cli_common/utils.py

```diff
@@ -215,14 +213,58 @@ def create_dask_client(args):
       empty string to start one locally
 
```

Comment:
@manthey Can you update this doc string with a brief description of the semantics of the values that can be passed for scheduler address?

Also since we are broadening the types of values it can take, maybe we should rename is to `scheduler` instead of `scheduler_address`. And change it accordingly in the XML specs of the CLIs that invoke this.